api/config/ingest/config_request.go

```diff
@@ -29,6 +29,7 @@ func DefaultConfigRequest() *ConfigRequest {
 	c.V1.Sys.Service.MaxNumberOfBundledRunMsgs = w.Int32(2500)
 	c.V1.Sys.Service.MaxNumberOfBundledActionMsgs = w.Int32(10000)
 	c.V1.Sys.Service.NumberOfRunMsgsTransformers = w.Int32(9)
+	c.V1.Sys.Service.NumberOfRunMsgPublishers = w.Int32(2)
```

Comment:
We are currently going to default to two publishers. This should be good for a local Elasticsearch.